Formula/retroshare.rb

```diff
@@ -0,0 +1,48 @@
+class Retroshare < Formula
+  desc "Friend to friend communication platform"
+  homepage "https://retroshare.cc/"
+  url "https://github.com/RetroShare/RetroShare/releases/download/v0.6.5/RetroShare-v0.6.5-source-with-submodules.tar.gz"
+  sha256 "d974d25cbcceccaf6e23e3b8d683d61e91b08c1853b3be0ecb06281809979cb3"
+  head "https://github.com/RetroShare/RetroShare.git"
+
+  depends_on "pkg-config" => :build
+  depends_on "miniupnpc"
+  depends_on "openssl"
+  depends_on "qt"
+  depends_on "rapidjson"
+  depends_on "sqlcipher"
+  depends_on "xapian"
+
+  def install
+    # Avoid RetroShare build system set wrong SDK version
+    inreplace "retroshare.pri",
+        "macx:CONFIG *= rs_macos10.11", ""
+    inreplace "retroshare-gui/src/retroshare-gui.pro",
+        "	target.path = \"$${BIN_DIR}\"",
+        "	target.path=#{prefix}"
+    system "qmake",
```

Comment:
This seems like a lot of configuration and things like the version number can very easily break. Can't upstream ship a saner default or allow something like `OPENSSL_DIR=#{Formula["xapian"].opt_include}`?

Reply:
Upstream changes may be done for next RetroShare release, so in the meantime isn't this acceptable?

Reply:
No, on both this and the other comment. People very rarely change a formula, other than version updates, unless something breaks. If it's not in a maintainable state before merging it it's probably never going to be in such a state. At the very least fixes for this need to be accepted by upstream, even if they're not in a tagged release yet.